simmechanics/data/icub3/ICUB_3_all_options.yaml

```diff
@@ -743,4 +743,59 @@ XMLBlobs:
             <gazebo reference="r_wrist_yaw">
               <implicitSpringDamper>1</implicitSpringDamper>
             </gazebo>
+    - |
+            <gazebo reference="r_upper_arm">
+              <selfCollide>true</selfCollide>
+              <collision>
+                <origin xyz="0.030743125543818514 0.141531075879301 0.06954757274337188" rpy="-0.0715644603308 0.0619699233851 -1.84623504024"/>
+                <geometry>
+                  <mesh>
+                    <scale>0.001 0.001 0.001</scale>
+                    <uri>model://iCub/meshes/simmechanics/sim_icub3_r_upperarm_prt.stl</uri>
+                  </mesh>
+                </geometry>

```

Comment:
Are you sure that it is necessary to duplicate this infor from the model manually? If you see how we inject collision info for `r_foot_rear`, this does not seems to be necessary, as  in the `collision` tag only the elements that are added need to be included. This avoids the need for manually updating the `origin` tag if the transform location changes.

Reply:
Indeed we don't need the duplicated section, I have removed it, tested it and it works properly, I have committed and updated the PR (waiting for tests)